requirements.txt

```diff
@@ -0,0 +1,14 @@
+# requirements for the various test suites
+
+pytest>=6.1.1,<7.0.0
+pytest-benchmark>=4.0.0,<5.0.0
+pytest-timeout>=2.1.0,<3.0.0
+ledgercomm>=1.1.0,<1.2.0
+ecdsa>=0.16.1,<0.17.0
+typing-extensions>=3.7,<4.0
+embit>=0.7.0,<0.8.0
+mnemonic==0.20
+bip32>=3.4,<4.0
+speculos>=0.12.0,<0.13.0
+ragger[speculos, ledgerwallet]>=1.6.0
```

Comment:
Be aware that the previous `speculos<0.13.0` constraint will be lifted.

Reply:
Also if you want to merge another dependency just like you did with Speculos in Ragger, you can use the `ragger[all_backends]` dependency, which pulls `speculos`, `ledgerwallet` and `ledgercomm`

Reply:
Thanks, I'll keep it in mind. Since this is only used in tests, not too urgent - I'll reconsider if needed.